templates/sysdig-cloud/1/rancher-compose.yml

```diff
@@ -0,0 +1,34 @@
+.catalog:
+  name: "Sysdig Cloud"
+  version: "v1"
+  description: "Container-Native Application and Infrastructure Monitoring"
+  minimum_rancher_version: v1.0.1
+  uuid: sysdig-cloud-1
+  questions:
+    - variable: "SDC_ACCESS_KEY"
+      label: "Sysdig Cloud access key"
+      description: "Your unique Sysdig Cloud access key - register for a Sysdig Cloud account at www.sysdig.com to receive a key."
+      type: "string"
+      required: true
+    - variable: "SDC_TAGS"
+      label: "Sysdig Cloud tags"
+      description: "Tags to be applied to all hosts on which the Sysdig Cloud container is deployed - these will surface in the Sysdig Cloud app. Should be a comma-separated list in the form of TAG_NAME:TAG_VALUE. For example: role:webserver,location:europe."
+      type: "string"
+      required: false
+    - variable: "VERSION"
+      label: "Sysdig Cloud version"
+      description: "Specify a version of the Sysdig Cloud container to pull (default will pull latest stable version)."
+      type: "string"
+      default: "latest"
+      required: true
+    - variable: "HOST_EXCLUDE_LABEL"
+      label: "Host exclude label"
+      description: "Specify a Rancher host label here that can be used to exclude deployment of the Sysdig Cloud container on any given host. Eg: sysdig.exclude_sysdigcloud=true (you could then add the label 'sysdig.exclude_sysdigcloud=true' to any Rancher host to exclude Sysdig Cloud from that host). To filter on any of multiple labels, use a comma-separated list."
+      type: "string"
+      default: sysdig.exclude_sysdigcloud=true
+      required: true
+    - variable: "SDC_ADDITIONAL_CONF"
+      label: "Additional Configuration"
+      description: "Content must be valid yaml code, use the escape \n parameter for new lines and adhere to proper indent requirements."
+      type: "string"
+      required: false
```

Comment:
# Line 35 to the end
++    - variable: "log_size"
++      label: "Max log size"
++      description: "Max log size"
++      type: "string"
++      default: 100mb
++      required: true